src/components/Security/Users/EditCustomMapping.vue

```diff
@@ -47,50 +85,72 @@
  */
 
 import Headline from '../../Materialize/Headline'
-import Mapping from './Steps/Mapping'
+import JsonEditor from '../../Common/JsonEditor'
 import {
   getMappingUsers,
   updateMappingUsers
 } from '../../../services/kuzzleWrapper'
+import { omit } from 'lodash'
```

Comment:
To avoid sending the whole lodash package to the client, it's better practice to import it as follow
`import omit from 'lodash/omit'`